engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -25,5 +25,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
   }
-
+  .description {
```

Comment:
Any preference?

Could make it content_view_version_description; however, that feels very specific.